code/modules/research/stock_parts.dm

```diff
@@ -328,3 +328,94 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	icon = 'icons/obj/stock_parts.dmi'
 	icon_state = "capacitor"
 	desc = "A debug item for research."
+
+
+//Mysterious machine components made by an unknown source. Designed to be doled out as an ultra-rare loot item.
+/obj/item/stock_parts/capacitor/quantumcap
```

Comment:
Why is this not documented in any capacity.

Reply:
This is being marked as DNM until I have the time to go over this entire PR

Reply:
they're not currently aquirable ingame (outside of a rare portal drop, as common) but I'm planning to use them as part of salvaging 2.
It seems like I failed to save the PR changelog changes with quantum, though

Reply:
added to the main PR body